src/snaphu/_unwrap.py

```diff
@@ -314,6 +318,10 @@ def unwrap(  # type: ignore[no-untyped-def]
         errmsg = f"init method must be in {init_methods}, instead got {init!r}"
         raise ValueError(errmsg)
 
+    # `phase_grad_window` must be a pair of positive integers.
+    phase_grad_window = tuple(phase_grad_window)
+    check_2d_shapes(phase_grad_window=phase_grad_window)
+
```

Comment:
Do we know if SNAPHU has any specific requirement on this window? Does the window need to be square or it can also be rectangular?

Reply:
Internally, SNAPHU has [a check](https://github.com/gmgunter/snaphu/blob/5981e65f3c267906b32dd5411bc27e0c4844f216/src/snaphu_io.c#L854-L859) that ensures that both window dimensions are greater than 0, but I think that's the only requirement. There isn't any constraint that forces the window to be square (just tested this locally to double-check).